src/app/loadout-builder/LockArmorAndPerks.tsx

```diff
@@ -170,6 +183,12 @@ function LockArmorAndPerks({
     _.sortBy(items, (i: LockedItemCase) => order.indexOf(i.bucket.hash))
   );
 
+  const modOrder = [...Object.values(Armor2ModPlugCategories), 'seasonal'];
+  const flatLockedArmor2Mods: LockedArmor2Mod[] = _.flatMap(
+    modOrder,
```

Comment:
Prefer `modOrder.flatMap` to `_.flatMap`

Reply:
Done